build.rs

```diff
@@ -60,7 +60,10 @@ fn main() {
                 }
             }
             Err(e) => {
-                println!("cargo:warning=Could not find zlib include paths via pkg-config: {}", e)
+                println!(
```

Comment:
Could you undo this formatting change as it's unrelated?
*CI currently doesn't validate the format, and it seems `cargo fmt` skips `build.rs` entirely.*

Reply:
Reverted! FYI, this change was made by running `cargo fmt`.